roles/common/tasks/main.yml

```diff
@@ -161,16 +161,17 @@
 
 - block:
   - name: drop an motd with ursula metadata for rhel
-    template:
-      src: etc/update-motd.d/90-ursula-motd
-      dest: /etc/90-ursula-motd
-      mode: 0755
-
-  - name: run the ursula-motd script from profile.d
-    template:
-      src: etc/profile.d/ursula-motd.sh
-      dest: /etc/profile.d/ursula-motd.sh
-      mode: 644
+    blockinfile:
```

Comment:
This will also write  marker in /etc/motd and will display that on screen as well. Which is ok, just mentioning.